test/services/open_banking_connector_spec.rb

```diff
@@ -21,25 +23,59 @@ class FetchMovementsTest < ActiveSupport::TestCase
 
       ::Nordigen::NordigenClient.stub(:new, @client_mock) do
         exception = assert_raises(StandardError) do |e|
-          OpenBankingConnector.new.fetch_movements(external_account_id: "1")
+          OpenBankingConnector.new.fetch_movements(account: @account_instance)
         end
 
         assert_equal exception.message, error.to_json
       end
     end
 
-    test "returns the main balance of the given existent account as a float number, the currency and the transactions" do
+    test "When no moments are present, then returns the main balance of the given existent account as a float number, the currency and the transactions for last 4 months" do
```

Comment:
There's a slight typo here, instead of moments it should be movement, would you mind amending it?